src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,59 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String email;
```

Comment:
```suggestion
@Column(unique = true)
    private String email;
```
В линке common mistakes есть пунктик о том что это поле должно быть уникальным, я знаю о аннотации @Column(unique = true) которая может помочь в этом)